prac_08/squaring.kv

```diff
@@ -0,0 +1,21 @@
+BoxLayout:
+    orientation: 'vertical'
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput:
+            id: input_number
+            text: '7'
+            font_size: 48
+            multiline: False
+        Button:
+            text: 'Square'
+            on_press: app.handle_calculate(input_number.text)
+        Label:
+            id: output_label
+            font_size: 48
+            color: (1, 0.7, 0.7, 1)
+    BoxLayout:
```

Comment:
I think that no need to write BoxLayout again

Reply:
Noted! Thank you